test/integration/agent/openflow_test.go

```diff
@@ -77,13 +75,13 @@ type testConfig struct {
 }
 
 func TestConnectivityFlows(t *testing.T) {
-	c = ofClient.NewClient(br)
-	err := ofTestUtils.PrepareOVSBridge(br)
+	c := ofClient.NewClient(openflowTestBrName)
```

Comment:
did you really need to remove the global variable for this? It made the test code less verbose.

Reply:
I thought many functions shared the same variable which is in global may not be a good practice. But it looks like my changes are not too different from the origin, so, reverted.